index.js

```diff
@@ -0,0 +1,57 @@
+const cron = require('node-cron')
+const Logger = require('le_node')
+const Promise = require('bluebird');
+const fs = Promise.promisifyAll(require('fs'));
+const {exec} = require('child_process');
+const formatter = require('dateformat')
```

Comment:
are we using an external lib just to use it once? uh no :)

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString